src/main/java/com/umc/networkingService/domain/branch/service/BranchServiceImpl.java

```diff
@@ -1,19 +1,143 @@
 package com.umc.networkingService.domain.branch.service;
 
+import com.umc.networkingService.domain.branch.converter.BranchConverter;
+import com.umc.networkingService.domain.branch.dto.request.BranchRequest;
+import com.umc.networkingService.domain.branch.dto.response.BranchResponse;
 import com.umc.networkingService.domain.branch.entity.Branch;
+import com.umc.networkingService.domain.branch.entity.BranchUniversity;
+import com.umc.networkingService.domain.branch.execption.BranchHandler;
 import com.umc.networkingService.domain.branch.repository.BranchRepository;
+import com.umc.networkingService.domain.branch.repository.BranchUniversityRepository;
+import com.umc.networkingService.domain.university.entity.University;
+import com.umc.networkingService.global.common.enums.Semester;
 import com.umc.networkingService.global.common.exception.ErrorCode;
-import com.umc.networkingService.global.common.exception.RestApiException;
+import com.umc.networkingService.global.utils.S3FileComponent;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.multipart.MultipartFile;
 
+import java.util.List;
+import java.util.Optional;
+import com.umc.networkingService.global.common.exception.RestApiException;
 import java.util.UUID;
 
 @Service
 @RequiredArgsConstructor
-public class BranchServiceImpl implements BranchService{
+public class BranchServiceImpl implements BranchService {
 
     private final BranchRepository branchRepository;
+    private final BranchUniversityRepository branchUniversityRepository;
+    private final S3FileComponent s3FileComponent;
+
+    private static final String BRANCH_CATEGORY = "branch";
+    private static final Integer PAGE_SIZE = 10;
+
+    @Transactional         //지부 생성
+    public UUID postBranch(BranchRequest.PostBranchDTO request) {
+
+        //이름이 없는 경우
+        if(request.getName() == null || request.getName().isEmpty()){
+            throw new BranchHandler(ErrorCode.BRANCH_NAME_EMPTY);
+        }
+        //설명이 없는 경우
+        if(request.getDescription() == null || request.getDescription().isEmpty()){
+            throw new BranchHandler(ErrorCode.BRANCH_DESCRIPTION_EMPTY);
+        }
+
+        Branch newBranch = BranchConverter
+                .toBranch
+                        (request
+                        ,uploadImageS3(BRANCH_CATEGORY,request.getImage())
+                        );
+
+        Branch savedBranch = branchRepository.save(newBranch);
+        if (savedBranch.getId() == null) {
+            throw new BranchHandler(ErrorCode.BRANCH_SAVE_FAIL);
+        }
+
+
+        return branchRepository.save(newBranch).getId();
```

Comment:
54번줄 코드에서 레포지토리에 save 메서드를 이용하여 저장한 객체를 반환했기 때문에, 한번 더 save 메서드를 호출하는 것보다, 반환된 객체를 사용하는 것이 더 좋을 거 같아요!!
```suggestion
        return savedBranch.getId();
```